source/openpower/keys.rst

```diff
@@ -29,8 +29,21 @@ On a simple level, instances can be accessed using ``ssh``::
 
 .. note::
 
-  <image_name> changes after rebuilding an instance with a different OS. Ubuntu machines -> ``ubuntu@<IP>``
-  CentOS machines -> ``centos@<IP>``, Debian machines -> ``debian@<IP>``, Fedora machines -> ``fedora@<IP>``, etc...
+  ``<image_name>`` changes after rebuilding an instance with a different OS.
+
+  +----------+--------------+
+  | OS       | Login Name   |
+  +==========+==============+
+  | Debian   | ``debian``   |
+  +----------+--------------+
+  | CentOS   | ``centos``   |
+  +----------+--------------+
+  | Ubuntu   | ``ubuntu``   |
+  +----------+--------------+
+  | Fedora   | ``fedora``   |
+  +----------+--------------+
+  | OpenSUSE | ``opensuse`` |
+  +----------+--------------+
```

Comment:
You might use an in-line csv-table instead of this method for creating a table. We have a few examples in the repo.